src/components/index.js

```diff
@@ -5,32 +5,34 @@ import {
   AxesRange,
   AxesSelector,
   CanvasSize,
-  ContourNumeric,
   ColorPicker,
+  ColorscalePicker,
+  ContourNumeric,
```

Comment:
Which ESLint setting are you using that reorders imports like this? I'll turn it on myself so we get less churn like this where your editor fixes my mistakes in your commits :)